src/groundlight/experimental_api.py

```diff
@@ -9,23 +9,26 @@
 import json
 from typing import Any, Dict, Union
 
-from groundlight_openapi_client.api.images_api import ImagesApi
+from groundlight_openapi_client.api.actions_api import ActionsApi
+from groundlight_openapi_client.api.image_queries_api import ImageQueriesApi
 from groundlight_openapi_client.api.notes_api import NotesApi
-from groundlight_openapi_client.api.rules_api import RulesApi
 from groundlight_openapi_client.model.action import Action
-from groundlight_openapi_client.model.condition import Condition
-from groundlight_openapi_client.model.note_creation_input import NoteCreationInput
-from groundlight_openapi_client.model.rule_creation_input import RuleCreationInput
-from model import Channel, Detector, PaginatedRuleList, Rule, Verb
+from groundlight_openapi_client.model.action_request import ActionRequest
+from groundlight_openapi_client.model.channel_enum import ChannelEnum
+from groundlight_openapi_client.model.condition_request import ConditionRequest
+from groundlight_openapi_client.model.note_request import NoteRequest
+from groundlight_openapi_client.model.rule_request import RuleRequest
+from groundlight_openapi_client.model.verb_enum import VerbEnum
+from model import Detector, PaginatedRuleList, Rule
```

Comment:
Any idea why this is still considered Experimental? Should we change that?

Reply:
At some point this moves into the main client body, but I think we just haven't had the impetus to change this yet. This was our way of leaving a door open in case we wanted to change things later